src/services/LoginRequest.js

```diff
@@ -0,0 +1,36 @@
+import Cookies from 'js-cookie';
+
+async function LoginRequest(credentials) {
```

Comment:
Only class names start with capital letters and you are not using `await` in function no need to make it `async` 
```suggestion
function loginRequest(credentials) {
```